### ISSUE TITLE
View PDF in waterfox using another PDF viewer plugin

### ISSUE THREAD

**Issue by Karllinst** [BrowserWorks/waterfox#483]
The option for using another PDF plugin (PDF-XChange Editor) in waterfox is missing in Options - Applications.
The PDF-XChange Editor NPAPI plugin was installed, but the option is disappeared.

The temporary solution is installing a older version waterfox (maybe waterfx 53), then enabled the option to use PDF-XChange Editor plugin view PDF in waterfox and updated the waterfox. 

Can you add the option back?

**Comment by atomGit** [comment]
i could be entirely wrong, but...

i don't think you need the plugin installed - try disabling/removing it, restart WF, then in Applications, set the PDF mime type to 'always ask' and see if you can point WF to your PDF viewer of choice after loading a [PDF doc](https://www.superteacherworksheets.com/reading-comp/2nd-kitten-play_KITTE.pdf)

**Comment by Karllinst** [comment]
@atomGit 
Thanks for your reply.
But what I mean is opening PDFs **in the tabs of WF**, like this:
https://i.imgur.com/fbbir4M.png

Otherwise, it just download the PDF files, then open it by the PDF viewer.

Therefore, I thought, to open a PDF file in browser, the plugin is essential.

**Comment by grahamperrin** [comment]
Just checking, there are two products at <https://www.tracker-software.com/product/pdf-xchange-editor>: 

1. PDF-XChange Editor
2. PDF-XChange Editor Plus

You use the first of those two, yes?

**Comment by Karllinst** [comment]
@grahamperrin 
Yes, the first one.

**Comment by grahamperrin** [comment]
Cross reference https://www.reddit.com/r/waterfox/comments/87cmcu/-/dwyx350/?context=3

**Comment by nummacway** [comment]
Right after I change said setting, random things happen sometimes, for example:
- When I set a freshly-installed Waterfox 56 to open PDFs with Adobe Reader DC, it previewed one with PDF-XChange Viewer inside the tab, something that apparently can't be newly set in Version 56.
- When I set it to preview (pdf.js), it downloaded the file.

I would love to see that setting fixed, as pdf.js is rather useless.
